src/main/java/smartthings/migration/BulkMigrationHandler.java

```diff
@@ -0,0 +1,42 @@
+package smartthings.migration;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import smartthings.cassandra.CassandraConnection;
+import smartthings.util.Util;
+
+import java.io.File;
+import java.util.Map;
+
+public class BulkMigrationHandler implements Handler {
```

Comment:
Why different from improving migration handler

Reply:
In some cases, old behavior could be faster.

For example, if someone had migrations in the table that were old and no longer run -- like creating then deleting a table or something like that -- which they had since removed from  the repo. In this case, the old method could be potentially faster for a small number of desired migrations.

I'll admit that is unlikely, optimizing around what I am guessing is not a best practice, and probably a marginal performance increase if any. I can make the same changes too the old handler if this seems like it adds unnecessary complexity.